pynetdicom/utils.py

```diff
@@ -183,3 +183,13 @@ def validate_uid(uid):
         return True
 
     return False
+
+
+def make_target(target_fn):
+    if _config.PASS_CONTEXTVARS:
+        if sys.version_info[:2] < (3, 7):
+            raise RuntimeError("PASS_CONTEXTVARS requires Python >=3.7.")
+        from contextvars import copy_context
```

Comment:
suggestion (non-blocking): Maybe handle an import error here instead of checking the version?

```python
try:
    from contextvars import copy_context
except ImportError as error:
    raise RuntimeError("PASS_CONTEXTVARS not supported on this Python version") from error
```